### ISSUE TITLE
Changing the sort order for all users for the "Song Titles" section?

### ISSUE THREAD

**Issue by Cutters1982** [ampache/ampache#1910]
Hello!

I wanted to know if it is at all possible to change the sort order for the "Song Titles" section for all users so that it sorts by artist then by title, as it doesn't seem to be doing that at present

if I press on the word "artist" it obviously puts a-z artist order, but the titles of the tracks are not a-z as well, for example:-

Unfinished Symphony (Original Mix) comes before Unfinished Symphony (Arc In The Sky Remix) under the "Song title" section.

Hope someone can assist with this!

**Comment by lachlan-00** [comment]
That's actually a good point.

When you sort by artist it orders the songs in id number second which isn't alphabetical, it's the order it was imported so it is a bad thing to do.

**Comment by Cutters1982** [comment]
Yeah, it's unfortunate that you can't have various sort levels like it was excel or something, surely it must be added, sort by artist then sort by title, for example.

**Comment by lachlan-00** [comment]
It depends on how the jquerymediatable handles things but if it's set by sql i think i should be able to extend this without too much trouble

**Comment by Cutters1982** [comment]
excellent, thanks, hope you can figure it out and allow it in the drop down menu
